core/domain/draft_upgrade_services_test.py

```diff
@@ -157,15 +198,12 @@ def test_convert_states_v32_dict_to_v33_dict(self):
                         'value': 1
                     }
                 }
-            })]
-        expected_draft_change_list = (
-            draft_upgrade_services.DraftUpgradeUtil._convert_states_v32_dict_to_v33_dict(  # pylint: disable=protected-access,line-too-long
-                draft_change_list))
-        self.assertEqual(
-            expected_draft_change_list[1].to_dict(),
-            draft_change_list[1].to_dict())
-        self.assertEqual(
-            expected_draft_change_list[0].to_dict(),
+            })
+        ]
+        # Version 33 adds a showChoicesInShuffledOrder bool, doesn't impact
```

Comment:
doesn't --> which doesn't 

(in general, always write in full prose sentences. It makes things easier for readers.)